app/sprinkles/core/src/Util/CheckEnvironment.php

```diff
@@ -351,36 +364,39 @@ public function checkPermissions()
 
     /**
      * Check that PHP meets the minimum required version.
+     *
+     * @return bool True if problem(s) found.
      */
-    public function checkPhp()
+    public function checkPhp(): bool
     {
-        $problemsFound = false;
-
-        // Check PHP version
-        if (version_compare(phpversion(), \UserFrosting\PHP_MIN_VERSION, '<')) {
-            $problemsFound = true;
+        try {
+            VersionValidator::validatePhpVersion();
+        } catch (VersionCompareException $e) {
             $this->resultsFailed['phpVersion'] = [
                 'title'   => "<i class='fas fa-code fa-fw'></i> You need to upgrade your PHP installation.",
```

Comment:
We limit the upper bound as well (PHP 8 would register as an incompatible version). Message should be updated to reflect this.

Reply:
Good point

Reply:
Updated the title to reflect the failure one